cargo-pgx/README.md

```diff
@@ -104,6 +104,11 @@ As shown by the screenshot above, it downloads the latest versions of Postgres v
 
 `pgx` is designed to support multiple Postgres versions in such a way that during development, you'll know if you're trying to use a Postgres API that isn't common across all versions. It's also designed to make testing your extension against these versions easy. This is why it requires you to have all fully compiled and installed versions of Postgres during development.
 
+In cases when default ports pgx uses to run PostgreSQL within are not available, one can specify
+custom values for these during initialization using `--base-port` and `--base-testing-port`
+options. One of the use cases for this is using multiple installations of pgx (using `$PGX_HOME` variable)
+when developing multiple extensions at the same time.
+
```

Comment:
@yrashk I think it'd be good to also mention here that they can be later changed by hand-editing `$PGX_HOME/config.toml`.

Once you do that, this will be good to merge.

Thanks a bunch!

Reply:
@eeeebbbbrrrr done